src/Uno.UI/UI/Xaml/UIElement.Layout.netstd.cs

```diff
@@ -115,6 +115,7 @@ public void Arrange(Rect finalRect)
 			{
 				LayoutInformation.SetLayoutSlot(this, finalRect);
 				HideVisual();
+				_isArrangeValid = true;
```

Comment:
@jeromelaban is this a valid change? Without this, I was getting a layout loop, as arrange kept reporting dirty and nothing was changing.

Reply:
LGTM, but add a test that validates this scenario using `UIElement.IsMeasureDirty`, so it does not regress later.